### DISCUSSION TITLE
Shared Expressions in TOM Explorer Grouped

### DISCUSSION THREAD

**Discussion by InsightABI** [TabularEditor/TabularEditor3#517] [Ideas]
I thought it would be more intuitive (and a space saver) if the Shared Expressions were grouped in folders, inside of the Shared Expressions folder, by their Query Group property (like they are in Power Query).  

![SharedExpressionFolders](https://user-images.githubusercontent.com/84812245/164075679-79ef46dd-caac-4cef-b89b-6bc699bd2e1a.png)

**Comment by otykier** [comment]
This has been discussed in #436 - TLDR, we’re considering it as an option.

**Reply by InsightABI** [reply]
I read it and I think that having the (3) options you proposed is the best approach.   Thanks!
